ASM/src/drop_overrides/obj_mure3.asm

```diff
@@ -0,0 +1,39 @@
+;Hacks for rupee towers to drop flagged collectibles
```

Comment:
What exactly is a rupee tower? I went to the actor on decomp to try for clarification and found none, lol

Reply:
This is referring the the actor Obj_Mure3. It's the actor that spawns a pattern of rupees. The term "tower" is what SceneNavi calls them (presumably because the first pattern option, which is unused in vanilla I believe, is a vertical line, aka a tower).

Reply:
Ah gotcha. Yeah the term tower just confused me.